src/mumble/ClientUser.cpp

```diff
@@ -190,6 +193,13 @@ void ClientUser::setLocalIgnore(bool ignore) {
 	emit muteDeafStateChanged();
 }
 
+void ClientUser::setLocalIgnoreTTS(bool ignoreTTS) {
+	if (bLocalIgnoreTTS == ignoreTTS)
+		return;
+	bLocalIgnoreTTS = ignoreTTS;
+	emit muteDeafStateChanged();
```

Comment:
I think you're misusing this signal here. After all TTS does have nothing to do with muting or deafening a user...

What is it that you want to achieve by emitting this signal here?

Reply:
Hmm, actually this method doesn't need to emit any signal. :thinking: I'll remove it.